### ISSUE TITLE
Tensorboard add image with boxes, labels, and confidence scores.

### ISSUE THREAD

**Issue by ayman-saleh** [pytorch/pytorch#27300]
## 🚀 Feature

Add a method to `torch.utils.tensorboard.writer.py` that will take an image, list of bounding boxes, list of class ids, and list of confidence scores and adds a summary to the events file.
## Motivation

In the TensorFlow Object Detection API, there is the functionality to add this, and it has been useful for visually profiling the performance of object detection models.

## Pitch

I would like to add this method into the `writer.py` file.

## Additional context

Here is an example of this in tensorboard.
![image](https://user-images.githubusercontent.com/30412615/66153079-d7f76980-e5e8-11e9-9792-d3ea478de86a.png)

**Comment by lanpa** [comment]
https://github.com/pytorch/pytorch/blob/b10929a14a32c90ac5943cc3f27a3a9105e6f53f/torch/utils/tensorboard/writer.py#L603 Hi @austinmw , would this function work for you?
